### ISSUE TITLE
What are the core packages

### ISSUE THREAD

**Issue by jpiaskowski** [bbolker/mixedmodels-taskview#6]
Starting a list here so we can keep track of this. Core packages have to be on CRAN (which sadly, leaves out broom.mixed)

lme4
nlme
brms
MCMCglmm
multilevelmod
geepack
lavaan

FYI: I found out the hard way that every time a core package is mentioned in the task view, it has to say include the tag `priority = "core"` to be processed correctly by `ctv::ctv2html()`. I tried to fixed the instances of this in the TV, but I may have missed some.

**Comment by bbolker** [comment]
Note that broom.mixed **is** on CRAN ... https://cran.r-project.org/web/packages/broom.mixed/index.html

I agree with everything on the list above. I *might* include  performance and DHARMa (for model checking, although arguably not since they're not MM-specific); lmerTest (for LMM p-values); glmmTMB (broader range of response distributions, faster in many cases) blme, rstanarm (Bayes), ordinal, robustlmm (the most important of the 'special types', for my work ...)

Maybe the best thing would be to decide approximately how many core packages we want in total? (20?)

**Comment by jpiaskowski** [comment]
Good news about broom.mixed! I missed that. All those ideas sound great. There's a function in the ctv package to install all "core" packages, so keeping it to 15-20 sounds reasonable.

**Comment by bbolker** [comment]
The only one of these I'm not familiar with is `multilevelmod` (I've installed it and see what it is, I think I see why it would be included -- parallel to `broom.mixed`). I think I can see the argument/case for all the rest. (`nlme` might not count against the total as it's present in the default installation.)

**Comment by jpiaskowski** [comment]
yeah, currently {multilevelmod} is limited in scope, but I have high hopes that it will eventually support more functionality since it's backed by the tidymodels team.

**Comment by jpiaskowski** [comment]
I'm personally not in favor of adding emmeans as a core package because it doesn't do mixed modelling - it doesn't even provide BLUPs (while {marginaleffects} does). However, I don't feel that strongly about this.

**Comment by palday** [comment]
I also wouldn't add `emmeans` as core -- its functionality is very general and not at all focused on mixed models / specialized to take advantage of what mixed models can do.
